templates/_pending.txt

```diff
@@ -5,7 +5,7 @@
 {{t "A while ago you created a Firefox Account, but never verified it. Click the link below to activate your account." }}
 
 # post verify
-{{t "You successfully connected your first device to your Firefox Account. Now you can sign in to Sync from your other devices, including Firefox for <a href='%(androidLink)s'>Android</a> and <a href='%(iosLink)s'>iOS</a>." }}
-{{t "You successfully connected your first device to your Firefox Account. To make the most of Sync, connect any <a href='%(androidLink)s'>Firefox for Android</a> or <a href='%(iosLink)s'>Firefox for iOS</a>."}}
+{{t "You successfully connected your first device to your Firefox Account. Now you can sign in to Sync from your other devices, including Firefox for <a href='%(androidUrl)s'>Android</a> and <a href='%(iosUrl)s'>iOS</a>." }}
+{{t "You successfully connected your first device to your Firefox Account. To make the most of Sync, connect any <a href='%(androidUrl)s'>Firefox for Android</a> or <a href='%(iosUrl)s'>Firefox for iOS</a>."}}
```

Comment:
Do these string changes require an explicit update to https://github.com/mozilla/fxa-content-server-l10n/pull/68/ before it gets merged?